### DISCUSSION TITLE
Stream the tv

### DISCUSSION THREAD

**Discussion by Idealsound85** [AndreyPavlenko/Fermata#171] [Q&A]
Hi Andrey,

I have a question for the stream tv. I add the paly list with the chanels but after around 1 min watching then is off. And i must push one more this same chanel. When i try this list looking in the VLC player and choose the loop. Then the tv alwyas continues.

Could you add the loop button for the tv player. Or maybe you got solution, what  i must set.

Best regads.

**Comment by AndreyPavlenko** [comment]
Hello,

I'm not aware of the loop option in vlc. What does it do?

**Comment by Idealsound85** [comment]
Hi, Im not mean about VLC. But for frrmata auto. The loop button. After that the fermata alaways bufor the signal. Becouse when i on the tv with big bitrate the fermata turn off the tv.Understand me??Wysłano z mojego urządzenia Galaxy
-------- Oryginalna wiadomość --------Od: Andrey ***@***.***> Data: 05.06.2022  11:31  (GMT+01:00) Do: AndreyPavlenko/Fermata ***@***.***> DW: Idealsound85 ***@***.***>, Manual ***@***.***> Temat: Re: [AndreyPavlenko/Fermata] Stream the tv (Discussion #171) 
Hello,
I'm not aware of the loop option in vlc. What does it do?
